FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Maksim Khozyashev on 09.09.2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+private enum Constants {
+	static let ok_200 = 200
+}
+
+final class FeedImageMapper {
+	struct Root: Decodable {
+		private let items: [Image]
+
+		var images: [FeedImage] {
+			return items.map { $0.feedImage }
+		}
+	}
+
+	struct Image: Decodable {
```

Comment:
`Image` should be marked as private as this is also an implementation detail of the `FeedImageMapper` 

```suggestion
	private struct Image: Decodable {
```